src/js/components/award/AwardInfo.jsx

```diff
@@ -15,43 +14,13 @@ const propTypes = {
 
 export default class AwardInfo extends React.Component {
 
-    constructor(props) {
-        super(props);
-
-        this.getStatus = this.getStatus.bind(this);
-    }
-
-    componentWillReceiveProps(nextProps) {
-        if (nextProps.selectedAward !== this.props.selectedAward) {
-            this.getStatus(nextProps.selectedAward);
-        }
-    }
-
-    getStatus(award) {
-        const awardStart = moment(award.period_of_performance_start_date, 'YYYY-MM-DD');
-        const awardEnd = moment(award.period_of_performance_current_end_date, 'YYYY-MM-DD');
-        const current = moment();
-        let progress = "";
-        if (current.isBefore(awardStart)) {
-            progress = "Awarded";
-        }
-        else if (current.isAfter(awardEnd)) {
-            progress = "Complete";
-        }
-        else {
-            progress = "In Progress";
-        }
-        this.progress = progress;
-    }
-
     render() {
         return (
             <div>
                 <SummaryBar
                     {...this.props}
-                    awardStatus={this.progress}
                     selectedAward={this.props.selectedAward} />
-                <div className="wrapper">
+                <div className="main-content">
```

Comment:
Can you add an `id="main-content"` and change this from a `<div>` to `<main>` tag per [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) and also to match [the search page](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/components/search/SearchPage.jsx#L164)?